acct.bionimbus.org/portal/gitops.json

```diff
@@ -79,253 +93,329 @@
+            "name": "File Type"
+          },
+          {
+            "field": "data_format",
+            "name": "File Format"
+          },
+          {
+            "field": "data_category",
+            "name": "File Category"
+          }
+        ],
+        "nodeCountTitle": "Files",
+        "manifestMapping": {
+          "resourceIndexType": "subject",
+          "resourceIdField": "subject_id",
+          "referenceIdFieldInResourceIndex": "object_id",
+          "referenceIdFieldInDataIndex": "object_id"
+        },
+        "accessibleFieldCheckList": [
+          "project_id"
+        ],
+        "accessibleValidationField": "project_id",
+        "downloadAccessor": "object_id"
       },
-      "data_format": {
-        "chartType": "stackedBar",
-        "title": "File Format"
-      }
-    },
-    "filters": {
-      "tabs": [
+      "buttons": [
+        {
+          "enabled": true,
+          "type": "file-manifest",
+          "title": "Download Manifest",
+          "leftIcon": "datafile",
+          "rightIcon": "download",
+          "fileName": "file-manifest.json",
+          "dropdownId": "download"
+        },
         {
-          "title": "File",
-          "fields": [
-            "project_id",
-            "data_type",
-            "data_format",
-            "data_category",
-            "experimental_strategy"
-          ]
+          "enabled": true,
+          "type": "export-files-to-workspace",
+          "title": "Export to Workspace",
+          "leftIcon": "datafile",
+          "rightIcon": "download"
         }
-      ]
-    },
-    "table": {
-      "enabled": true,
-      "fields": [
-        "project_id",
-        "subject_submitter_id",
-        "file_name",
-        "data_format",
-        "data_type",
-        "data_category",
-        "experimental_strategy",
-        "file_size",
-        "submitter_id",
-        "object_id",
-        "md5sum"
-      ]
-    },
-    "guppyConfig": {
-      "dataType": "file",
-      "fieldMapping": [
-        { "field": "object_id", "name": "GUID" }
       ],
-      "nodeCountTitle": "Files",
-      "manifestMapping": {
-        "resourceIndexType": "subject",
-        "resourceIdField": "_subject_id",
-        "referenceIdFieldInResourceIndex": "object_id",
-        "referenceIdFieldInDataIndex": "object_id"
-      },
-      "accessibleFieldCheckList": ["project_id"],
-      "accessibleValidationField": "project_id",
-      "downloadAccessor": "object_id"
-    },
-    "buttons": [
-      {
-        "enabled": true,
-        "type": "file-manifest",
-        "title": "Download Manifest",
-        "leftIcon": "datafile",
-        "rightIcon": "download",
-        "fileName": "file-manifest.json"
-      }
-    ],
-    "dropdowns": {}
+      "dropdowns": {}
+    }
+  ],
+  "dataExplorerConfig": {
+    "terraExportURL": null,
+    "sevenBridgesExportURL": null
+  },
+  "fileExplorerConfig": {
+    "terraExportURL": null,
+    "sevenBridgesExportURL": null
```

Comment:
I don't know where r 410-419 come from

Reply:
I'm guessing this is harmless (the automation just assumes these parameters are part of a common template and every environment should have them (which is a bad assumption).
But I think Windmill would just ignore these values if they are null..
let me cc: @mfshao to double-check. (just in case).